Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1005,7 +990,8 @@ static int GetMenuIDFromHotkey(unsigned int key)
 
 void OnAfterLoadCallback()
 {
-  // warning: this gets called from the CPU thread, so we should only queue things to do on the
+  // warning: this gets called from the CPU thread, so we should only queue
+  // things to do on the
   // proper thread
```

Comment:
Bring multi line comments together if they're clang formatted to be this yukky.
So move `//` proper thread` up a line